Domains/Profiles/Profiles.ecschema.xml

```diff
@@ -366,4 +367,118 @@
     </ECEntityClass>
+
+    <ECEntityClass typeName="ProfiledSolidDefinitionElement" modifier="Abstract" displayLabel="Profiled Solid Definition Element" description="TODO">
+        <BaseClass>bis:DefinitionElement</BaseClass>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="PhysicalTypeUsesProfiledSolidDefinition" modifier="None" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="bis:PhysicalType"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="ProfiledSolidDefinitionElement"/>
+        </Target>
+        <ECProperty propertyName="IsMirrored" typeName="boolean" displayLabel="Is Mirrored" description="TODO" />
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="PhysicalElementUsesProfiledSolidDefinition" modifier="None" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="bis:PhysicalElement"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="ProfiledSolidDefinitionElement"/>
+        </Target>
+        <ECProperty propertyName="IsMirrored" typeName="boolean" displayLabel="Is Mirrored" description="TODO" />
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IProfiledSolidDefinition" modifier="Abstract" displayLabel="IProfiled Solid Definition" description="TODO">
+        <ECCustomAttributes>
+            <IsMixin xmlns='CoreCustomAttributes.01.00.00'>
+                <AppliesToEntityClass>ProfiledSolidDefinitionElement</AppliesToEntityClass>
+            </IsMixin>
+        </ECCustomAttributes>
+    </ECEntityClass>
+    
+    <ECRelationshipClass typeName="IProfiledSolidDefinitionRefersToProfile" modifier="Abstract" strength="referencing" strengthDirection="forward" description="TODO">
+        <BaseClass>bis:ElementRefersToElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="uses" polymorphic="true">
+            <Class class="IProfiledSolidDefinition"/>
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is used by" polymorphic="true">
+            <Class class="Profile"/>
+        </Target>
+        <ECProperty propertyName="CardinalPointName" typeName="string" displayLabel="Cardinal Point" description="TODO" category="ProfiledSolidDefinitionProperties" />
+        <ECProperty propertyName="Offset" typeName="point2d" displayLabel="Offset" description="TODO" category="ProfiledSolidDefinitionProperties" />
```

Comment:
How are categories being used? Do they group Elements properties in Design Review based on category? I'm not aware of any control that is able to view relationships, how would these categories be used with relationships?

Reply:
You're right that today, UX doesn't show most relationships or their properties but that can change in the future - either in Design Review, or some more specialized UX.